### ISSUE TITLE
[Feature Request] Add information for server updates to installation instructions

### ISSUE THREAD

**Issue by rsjaffe** [tonesto7/echo-speaks#290]
**Is your feature request related to a problem? Please describe.**
I was receiving _EchoApp (v3.6.2.0) | wakeupServerResp Exception: java.lang.Exception: No response data exists for async request_ errors without understanding why.

I didn't realize that the server needed updating until I searched the Hubitat help forums, then the server update process was convoluted (though it did work). Later, I found the instructions for automated server updates, which would have solved this issue easily.

To summarize:

- I didn't realize that the server needed updating
- The manual update process was convoluted

**Describe the solution you'd like**
Add to the installation instructions information about setting up automatic server updates (e.g., with a link to https://tonesto7.github.io/echo-speaks-docs/updates/server_update/) to help those with github accounts set up an automated process when they set up their initial install.
